middleware.ts

```diff
@@ -0,0 +1,10 @@
+import { NextResponse } from 'next/server';
+import type { NextRequest } from 'next/server';
+
+export function middleware(request: NextRequest) {
+  const url = request.nextUrl.clone();
+  if (url.pathname === '') {
+    url.pathname = '/';
+    return NextResponse.redirect(url);
+  }
+}
```

Comment:
clever

Reply:
We'll see if this actually fixes it haha 🙏